packages/wonder-blocks-modal/components/modal-launcher.test.js

```diff
@@ -181,4 +186,24 @@ describe("ModalLauncher", () => {
             "either 'children' or 'opened' must be set",
         );
     });
+
+    test("If backdropDismissEnabled set to false, clicking the backdrop does not trigger `onClose`", () => {
```

Comment:
This test probably should've been in the previous PR since that's when it was added.